catalyst/dl/callbacks/metrics/accuracy.py

```diff
@@ -4,6 +4,25 @@
 from catalyst.dl.utils import criterion
 
 
+def _get_list_args(num_classes: int) -> List[int]:
```

Comment:
maybe, `_get_default_accuracy_args`?

Reply:
no, because it used also in `MapKCallback`

Reply:
mAP is accuracy still

Reply:
we just need better naming, rather than such abstract one

Reply:
don't want to name it `accuracy`, because if we add some new classification callback this func will be useful

Reply:
you place it in the `metrics/accuracy.py` and make it protected, so it looks  okay to use it only for the accuracy

Reply:
ok